README.md

```diff
@@ -21,14 +21,14 @@ Please note that we are not taking external contributions for new test pages, bu
 
 ### Test domains
 
-We have couple of test domains, that all resolve to `privacy-test-pages.glitch.me`, which help us simulate various scenarios:
+We have couple of test domains, that all resolve to `privacy-test-pages.site`, which help us simulate various scenarios:
 
 - `www.first-party.site` - an alternative first-party domain used for tests that require first-party resources on other subdomains (e.g., `hsts.first-party.site`)
 - `good.third-party.site` - non-tracking third party, it's not on our blocklist and will not be blocked by our clients
 - `broken.third-party.site` - tracking third party that we can't block (e.g. due to brekage), it's on our blocklist, but it will not be blocked by our clients
 - `bad.third-party.site` - tracking third party that's on our blocklist and our clients will block
 
-We also have additional test domains that resolve to `ad-attribution-test-pages.glitch.me`, specifically for the Ad Attribution tests hosted [here](https://www.search-company.site):
```

Comment:
this is not right, this domain doesn't exist (ad-attribution-test-pages.site). We only had ad-attribution-test-pages.glitch.me because of glitch limitation, so perhaps we should reword it to:

> We also have additional test domains specifically for the Ad Attribution tests hosted [here] …

Reply:
Updated